### ISSUE TITLE
Remove .TIF extension form Load Data file extension filter (Trac #264)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#412]
I think we have inadvertently caused the confusion here: .TIF appears 
as one of the extension filters to the Load Data file chooser, along 
with .XML, .DAT, etc. But no other graphic format extensions are in 
this filter. It should be removed.

Steve

(Paul replied to Sonia on 31 May)

On Fri, May 30, 2014 at 2:33 PM, sonia letichevsky <sonialetiche@gmail.com> wrote:

Dear Sir/Madam,
I am a brazilian researcher and I recently installed SASVIEW in my 
computer. 

I noticed that despite the fact that when I press "load file", one 
of the extension avaiable for to choose is ".TIF" I could not 
sucessfully open any image saved as TIFF.

Could you help me to solve this problem, please?

Best Regards,
 Sonia

Migrated from http://trac.sasview.org/ticket/264
```json
{
    "status": "closed",
    "changetime": "2015-02-13T11:42:47",
    "_ts": "2015-02-13 11:42:47.228040+00:00",
    "description": "I think we have inadvertently caused the confusion here: .TIF appears \nas one of the extension filters to the Load Data file chooser, along \nwith .XML, .DAT, etc. But no other graphic format extensions are in \nthis filter. It should be removed.\n\nSteve\n\n(Paul replied to Sonia on 31 May)\n\nOn Fri, May 30, 2014 at 2:33 PM, sonia letichevsky <sonialetiche@gmail.com> wrote:\n\nDear Sir/Madam,\nI am a brazilian researcher and I recently installed SASVIEW in my \ncomputer. \n\nI noticed that despite the fact that when I press \"load file\", one \nof the extension avaiable for to choose is \".TIF\" I could not \nsucessfully open any image saved as TIFF.\n\nCould you help me to solve this problem, please?\n\nBest Regards,\n Sonia\n",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2014-05-31T12:47:27",
    "component": "SasView",
    "summary": "Remove .TIF extension form Load Data file extension filter",
    "priority": "major",
    "keywords": "TIF, File Extension",
    "milestone": "SasView 3.1.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2014/05/31 15:38:33`: **butler** changed milestone from "SasView Next Release +1" to "SasView 3.0.1"

**Comment by butlerpd** [comment]
Trac update at `2014/05/31 16:56:25`: **butler** commented:

Further investigation (and jogging of old memories) reveals that tif is indeed one of the registered readers.  This is DIFFERENT from the image viewer/converter in the tools menu. As far as I can tell it is NOT using matplotlib.  It is one of the dataloader readers (tiff_reader.py) and is registered (hence showing up in drop down).  However the reader function trys to import Image ... which fails ... not clear what "Image" that is.  It may be that years ago that was a matplotlib or wx image?

Also the history is that for 2D reading back in 2006-8 somewhere we were using the NIST 2D raw format.  Later some people emailed us about TIFF (their Xray data was a TIFF) and Mat implemented such a reader and the users were very happy.

Later as the philosphy for sasview gelled and we started making a clear and hard demarcation between raw and reduced data and insisting that only reduced data is general enough to be used for a generic analysis package, we changed the internal 2D data structures and started reading 2D reduced formats from NIST IGOR.  However, we thought we'd try to maintain some backward compatibility for users who got used to it.  At some point the TIFF stopped working or had problems as I recall but I cannot remember what and we decided to quit supporting TIFF.  That does not explain why it is here, but ....

Two options:
1) we can try to fix the TIFF reader here
2) we can remove the TIFF reader from the registry

I STRONGLY favor the 2nd as it is clear the TIFF has not worked in a while so no users should be used to it.  Keeping it would be problematic because it can only give pixel data with arbitrary intensity which is not in keeping with the "reduced data" concept and secondly cannot be fit since we don't know what the q scale is without some intervention.  The image converter (once it works) should be the correct approach.
